submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,367 @@
+
+## The Questions
+
+We begin with the question that started it all: 
+
+**(1) Does processing coal into solid fuel get more energy out of it?**
+
+We can convert the coal liquification outputs directly into solid fuel, but we also have cracking available, so: 
+
+**(2) How much of a difference does cracking make when making solid fuel?**
+
+Then we check how processing coal stands against the conventional way of making solid fuel: 
+
+**(3) How does oil processing compare with coal liquification when producing solid fuel?**
+
+But then we note that we don't have to stop the processing at solid fuel: 
+
+**(4) Does further processing solid fuel into rocket fuel get more energy out of it?**
+
+And then we consider going further and adding uranium: 
+
+**(5) Does further processing rocket fuel into nuclear fuel get more energy out of it?**
+
+Then we consider that uranium is already being used for a different fuel type and we look at how that compares: 
+
+**(6) How does burning nuclear fuel for power compare with making uranium fuel cells to "burn" in reactors instead?**
+
+After considering all the energy equations under default conditions, we next look at how things change upon introducing modules:
+
+**(7) What are the effects of modules on energy gains?**
+
+Finally, we take a step back and look at the big picture, noting other factors like costs and alternate technologies, and we ask: 
+
+**(8) How useful is fuel processing in general?**
+
+With all the conversion options in the vanilla game, this list of questions actually covers all of the burner fuels except for wood, which is not converted into anything unless you use a [charcoal mod]([Charcoal Burner - Factorio Mods](https://mods.factorio.com/mod/CharcoalBurner)). Hence, here is a quick note about wood as a fuel: You may end up with a lot of it as a byproduct of clearing space using an axe or robots. However, it runs out quickly because of how little energy it provides. Instead of cutting trees for extra fuel, you would probably benefit more from using the wood to make power poles or chests, or from keeping the trees around for pollution absorption. As for other fuels, since we use conversion diagrams to answer the questions, first let's look at how they were made.
+
+## Making Conversion Diagrams
+
+In the diagrams, the recipes for various fuel-related crafting processes are applied. The input and output ratios are preserved but we run each process fractionally to match the input quantities rather than aiming for full crafting cycles. Most of the used recipes can be checked quickly from [the wiki page for oil processing](https://wiki.factorio.com/Oil_processing). The energy values of the fuels are in-game constants, referenced in [the wiki page for fuels](https://wiki.factorio.com/Fuel). The energy consumed to heat steam to 165°C is referenced in [the wiki page for steam](https://wiki.factorio.com/Steam). 
```

Comment:
```suggestion
In the diagrams, the recipes for various fuel-related crafting processes are applied. The input and output ratios are preserved, but we run each process fractionally to match the input quantities rather than aiming for full crafting cycles. Most of the used recipes can be checked quickly from [the wiki page for oil processing](https://wiki.factorio.com/Oil_processing). The energy values of the fuels are in-game constants, referenced in [the wiki page for fuels](https://wiki.factorio.com/Fuel). The energy consumed to heat steam to 165°C is referenced in [the wiki page for steam](https://wiki.factorio.com/Steam). 
```